plugins/dig-now.cpp

```diff
@@ -531,6 +532,7 @@ static bool dig_tile(color_ostream &out, MapExtras::MapCache &map,
                     map.setTiletypeAt(pos_above,
                             get_target_type(tt, df::tiletype_shape::RAMP_TOP));
                     remove_ramp_top(map, DFCoord(pos.x, pos.y, pos.z+2));
+                    propagate_vertical_flags(map, DFCoord(pos.x, pos.y, pos.z + 1));
```

Comment:
now that we've handled channeling and ramps specifically, is there any reason to keep the `propagate_vertical_flags` call on line 556?

Reply:
I thought digging a wall with a upstairs below it would reveal a down stairs, but now that I tested and see that is not the case I don't know where I got the idea from.